From bb62917973315b28965e46182d79064324e17efb Mon Sep 17 00:00:00 2001
From: Stelios Tsampas <loathingkernel@gmail.com>
Date: Thu, 23 Apr 2026 00:51:43 +0300
Subject: [PATCH 09/14] Revert "HACK: shell32: Preserve duplicated
 Documents/Stuff directories"

This reverts commit 3979d95d1eb0da4485f3d311ae3c2ea2280d4837.
---
 dlls/shell32/shellpath.c | 17 -----------------
 1 file changed, 17 deletions(-)

diff --git a/dlls/shell32/shellpath.c b/dlls/shell32/shellpath.c
index 8985ecffea8..aab3f91c5cd 100644
--- a/dlls/shell32/shellpath.c
+++ b/dlls/shell32/shellpath.c
@@ -3147,23 +3147,6 @@ static HRESULT create_extra_folders(void)
         hr = SHGetFolderPathAndSubDirW(0, CSIDL_MYPICTURES | CSIDL_FLAG_CREATE, NULL,
                                        SHGFP_TYPE_DEFAULT, L"Screenshots", path);
     }
-
-
-    /* Proton HACK: In older Proton versions, duplicate Stuff directories were
-     * created at both %PROFILE%\Music and %PROFILE\Documents\Music. Due to
-     * some bugs when downgrading to those older Proton versions, create those
-     * missing Documents directories here, too. */
-    SHGetFolderPathAndSubDirW(0, CSIDL_PERSONAL | CSIDL_FLAG_CREATE, NULL,
-                                   SHGFP_TYPE_DEFAULT, L"Downloads", path);
-    SHGetFolderPathAndSubDirW(0, CSIDL_PERSONAL | CSIDL_FLAG_CREATE, NULL,
-                                   SHGFP_TYPE_DEFAULT, L"Music", path);
-    SHGetFolderPathAndSubDirW(0, CSIDL_PERSONAL | CSIDL_FLAG_CREATE, NULL,
-                                   SHGFP_TYPE_DEFAULT, L"Pictures", path);
-    SHGetFolderPathAndSubDirW(0, CSIDL_PERSONAL | CSIDL_FLAG_CREATE, NULL,
-                                   SHGFP_TYPE_DEFAULT, L"Templates", path);
-    SHGetFolderPathAndSubDirW(0, CSIDL_PERSONAL | CSIDL_FLAG_CREATE, NULL,
-                                   SHGFP_TYPE_DEFAULT, L"Videos", path);
-
     return hr;
 }
 
-- 
2.54.0

